### PR TITLE
ADEN-4183 Animate VUAP on Mercury.

### PR DESCRIPTION
**Pull request by drets**
* http://wikia-inc.atlassian.net/browse/ADEN-4183

### CODE REVIEW

File: front/main/app/styles/module/_ads.scss [hunk 1]
```diff
@@ -39,9 +46,14 @@ $closeButtonHeight: 40px;
 	width: 100%;
 
 	.ad-slot {
-		background: none;
+		background: #000;
```
Comment (nandy-andy) [inline front/main/app/styles/module/_ads.scss]: What happens if there is no background color set? I wonder if we could use here some pre-defined SCSS variable.

Comment (nandy-andy) [inline front/main/app/styles/module/_ads.scss]: How about [$color-black](https://github.com/Wikia/style-guide/blob/dev/src/scss/lib/_settings.scss#L114) used for example for [menu list text color](https://github.com/Wikia/mercury/blob/dev/front/main/app/styles/_settings.scss#L130)?

File: front/main/app/styles/module/_ads.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import 'ads-floor';
+// Use Wikia style guide settings
+@import '../../../bower_components/wikia-style-guide/src/scss/lib/settings';
```
Comment (bognix) [inline front/main/app/styles/module/_ads.scss]: are you sure you have to import that?
You should use colors from front/main/bower_components/design-system/dist/scss/wds-variables/_colors.scss and this file is already imported.

File: front/main/app/styles/module/_ads.scss [hunk 3]
```diff
@@ -31,6 +32,13 @@ $closeButtonHeight: 40px;
 	margin: 0 -20px;
 }
 
+.vuap-loaded {
+	transition: padding 0.4s ease;
+	-webkit-transition: padding 0.4s ease;
+	-moz-transition: padding 0.4s ease;
+	-o-transition: padding 0.4s ease;
```
Comment (bognix) [inline front/main/app/styles/module/_ads.scss]: no need for all the prefixes. http://caniuse.com/#feat=css-transitions

File: front/main/app/styles/module/_ads.scss [hunk 4]
```diff
@@ -39,9 +47,14 @@ $closeButtonHeight: 40px;
 	width: 100%;
 
 	.ad-slot {
-		background: none;
+		background: $color-black;
```
Comment (bognix) [inline front/main/app/styles/module/_ads.scss]: why the black color as background?

Comment (drets) [inline front/main/app/styles/module/_ads.scss]: https://github.com/Wikia/mercury/pull/2943#discussion_r88451142

Comment (bognix) [inline front/main/app/styles/module/_ads.scss]: name of black color in DS is `$wds-color-black: #1a1a1a;`. Are you sure color-black is going to work?

Comment (drets) [inline front/main/app/styles/module/_ads.scss]: y

File: front/main/app/styles/module/_ads.scss (hunk 1 above)
Comment (nandy-andy) [inline front/main/app/styles/module/_ads.scss]: As discussed in bigger group on Slack: doesn't make sense for background.

**Pull request merged**